src/handlers/pool/pool_controller.go

```diff
@@ -0,0 +1,84 @@
+package pool
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+)
+
+func (pool *PoolService) finishPool() http.HandlerFunc {
+
+	type PoolDetails struct {
+		DrawnNumbers   []int `json:"drawn_numbers"`
+		TwoMultiplier  int   `json:"two_multiplier"`
+		FiveMultiplier int   `json:"five_multiplier"`
+	}
+
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodPost {
+			http.Error(w, "Only POST method is allowed", http.StatusMethodNotAllowed)
+			return
+		}
+
+		activePool, err := pool.poolStore.GetTodayPool()
+		if err != nil {
+			http.Error(w, "No active pool", http.StatusBadRequest)
+			return
+		}
+
+		drawnNumbers, err := drawNumbers(8, 99)
```

Comment:
Can we move those as constants and adjust the indexes below to depend on the number of balls. That way we won't have to update them if we change the total number of balls.

Reply:
Done